### PR TITLE
blog: announce ctlptl

### PR DESCRIPTION
**Pull request by nicks**
Hello @ellenkorbes, @victorwuky, @maiamcc,

Please review the following commits I made in branch nicks/ch10091:

ba3370598b567282afa0cff4f57301bce23aa114 (2020-11-06 19:41:44 -0500)
blog: announce ctlptl

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

File: blog/_posts/2020-11-08-ctlptl.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+- memory
+
+If one of these is off (e.g., one dev accidentally set their CPU too low), the
+environment is awful and uncomfortable.
+
+A well-organized `ctlptl` will help to fix that.
+
+## Where do I start?
+
+Check out [ctlptl on Github](https://github.com/tilt-dev/ctlptl) for installation instructions
```
Comment (victorwuky) [inline blog/_posts/2020-11-08-ctlptl.md:66]:
```suggestion
Check out [ctlptl on GitHub](https://github.com/tilt-dev/ctlptl) for installation instructions
```

**Review by victorwuky — APPROVED** [review]
Great!

File: blog/_posts/2020-11-08-ctlptl.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+- a dev registry
+- storage
+- mounts
+- cpu
+- memory
+
+If one of these is off (e.g., one dev accidentally set their CPU too low), the
+environment is awful and uncomfortable.
+
+A well-organized `ctlptl` will help to fix that.
```
Comment (maiamcc) [inline blog/_posts/2020-11-08-ctlptl.md:62]: Maybe worth a line for:

> `ctlptl` helps a single dev easily set up their dev environment according to best practices; and it gives a Tilt maintainer a one-liner to give their team for easy cluster setup.

Or something like that--just to explicitly call out the diff personas.

**Review by maiamcc — APPROVED** [review]
🙌

**Comment by ellenkorbes** [comment]
Nitpick: On their [webpage](https://kind.sigs.k8s.io/) kind is always written all lowercase. Since that might be confusing to readers Kind is okay IMO. But having KIND as well is a bit much.

File: blog/_posts/2020-11-08-ctlptl.md (hunk 2 above)
Comment (nicks) [reply blog/_posts/2020-11-08-ctlptl.md:62]: i like it! done

**Comment by nicks** [comment]
ya, agreed - adjusted the capitalization a bit

**Pull request merged**